api/package.json

```diff
@@ -39,6 +39,13 @@
     "fpx": "bin/cli.js"
   },
   "dependencies": {
+    "@clack/core": "^0.3.4",
```

Comment:
this is an unfortunate hack - the api needs the same deps as the cli in order for the cli to work after its build is copied into the api's `dist` folder

we should probably move the cli folder inside the api, but i wanna save that for a future pr